go-controller/pkg/cluster/cluster.go

```diff
@@ -35,6 +35,8 @@ const (
 	OvnHostSubnet = "ovn_host_subnet"
 	// OvnClusterRouter is the name of the distributed router
 	OvnClusterRouter = "ovn_cluster_router"
+	// OvnNodeManagementPortMacAddress is the constant string representing the annotation key
+	OvnNodeManagementPortMacAddress = "ovn_node_management_mac_address"
```

Comment:
@girishmg does it really need to be shorter? it accurately describes what it does, and nobody is ever going to type this :) We tried variations on MgtPortMAC and other stuff, and abbreviations just end up looking ugly IMHO

Reply:
@dcbw ok. How about "ovn_node_mgmtport_mac_address"?

Reply:
@dcbw should we also create a namespace for our annotations? Say, `ovn.org`. Therefore, the annotations will become -- `ovn.org/host_subnet` or `ovn.org/node._mgmtport_mac_address`

Reply:
@girishmg sure, we should probably do this.

Reply:
And I'm fine with node_mgmtport_mac_address

Reply:
@girishmg if we're going to follow best practice upstream, it would actually be something like:

`k8s.v1.ovn.org/nodeMgmtPortMacAddress`

eg, `k8s` because this is related to Kubernetes, `v1` to be correctly versioned if we change things later, and camelCase for the annotation itself.

Reply:
Examples of annotations from the Kube source tree: https://gist.github.com/dcbw/c293ea9302f5a064f26b23f43bae135c

Reply:
Yeah, '-' is better. Also, '_' is mostly for programming.